doc/tutorials/enable_ivshmem.rst

```diff
@@ -9,63 +9,81 @@ solution or hv-land solution, according to the usage scenario needs.
 While both solutions can be used at the same time, VMs using different
 solutions cannot communicate with each other.
 
+Enable Ivshmem Support
+**********************
+
+The ``ivshmem`` solution is disabled by default in ACRN. You can enable
+it using the :ref:`ACRN configuration toolset <acrn_config_workflow>` with these
+steps:
+
+- Enable ``ivshmem`` via ACRN configuration tool GUI.
+
+   - Set ``IVSHMEM_ENABLED`` to ``y``
```

Comment:
@dbkinder , there is no way to refer to the config option automatically (just like we do with APIs using Doxygen), is there?

Reply:
yes, there is:  ``:option:`hv.FEATURES.IVSHMEM.IVSHMEM_ENABLED` ``

Reply:
I force-pushed an update to add references to the option documentation, and will approve and merge.